src/libraries/Microsoft.Extensions.Http/tests/Microsoft.Extensions.Http.Tests/Logging/FailedHttpRequestLoggingTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit;
+
+namespace Microsoft.Extensions.Http.Tests.Logging
+{
+    public class FailedHttpRequestLoggingTests
+    {
+        private const string OuterLoggerName = "System.Net.Http.HttpClient.test.LogicalHandler";
+        private const string InnerLoggerName = "System.Net.Http.HttpClient.test.ClientHandler";
+
+        private static class EventIds
+        {
+            public static readonly EventId RequestHeader = new EventId(102, "RequestHeader");
+            public static readonly EventId ResponseHeader = new EventId(103, "ResponseHeader");
+            public static readonly EventId RequestFailed = new EventId(104, "RequestFailed");
+
+            public static readonly EventId RequestPipelineRequestHeader = new EventId(102, "RequestPipelineRequestHeader");
+            public static readonly EventId RequestPipelineResponseHeader = new EventId(103, "RequestPipelineResponseHeader");
+            public static readonly EventId PipelineFailed = new EventId(104, "RequestPipelineFailed");
```

Comment:
I believe other ids are not used in this test

```suggestion
            public static readonly EventId RequestFailed = new EventId(104, "RequestFailed");
            public static readonly EventId PipelineFailed = new EventId(104, "RequestPipelineFailed");
```